components/pedigree/resources/src/main/resources/pedigree/hpoTerm.js

```diff
@@ -75,7 +75,7 @@ define([
     }
 
     HPOTerm.getServiceURL = function() {
```

Comment:
that is addressed in [PR for PT-1403](https://github.com/phenotips/phenotips/pull/2293/commits/4fb308633c7c1c356bd91b18c2c9d3e3634bd547)